### DISCUSSION TITLE
Version 8.2 Python Problem

### DISCUSSION THREAD

**Discussion by indiggojorge** [NativeScript/NativeScript#9929] [Q&A]
Hi. How can I get version 8.3? I know it's not released yet but I'd like to try it. For some reason, "ns doctor ios" in my project directory shows "WARNING: Couldn't retrieve installed python packages". I am running Mac OS 12.4. I followed the instructions to alias python, install pip and six. Still no success.

**Comment by rigor789** [comment]
Easiest way right now:

```bash
brew install python
sudo ln -s -f $(which python3) /usr/local/bin/python
```

Then it should work - ran into this myself, because the alias to python3 when using the system python prompted the XCode tools window every time, and didn't work...

**Reply by indiggojorge** [reply]
I followed your steps. Then, when I type "python" from the command line, I get this ...

python: error: Failed to locate 'python'.
xcode-select: Failed to locate 'python', requesting installation of command line developer tools.

"ns doctor ios" gives me this ...

✖ WARNING: Couldn't retrieve installed python packages. 
 We cannot verify your python installation is setup correctly. Please, make sure you have both 'python' and 'pip' installed.
